ansible/roles/docker/templates/alpine319.Dockerfile.j2

```diff
@@ -0,0 +1,62 @@
+FROM alpine:3.19
+
+ENV LC_ALL C
+ENV USER {{ server_user }}
+ENV JOBS {{ server_jobs | default(ansible_processor_vcpus) }}
+ENV SHELL /bin/bash
+ENV HOME /home/{{ server_user }}
+ENV PATH /usr/local/venv/bin:/usr/lib/ccache/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Comment:
Added venv

Reply:
Refs: https://github.com/nodejs/build/pull/3560

I put this one in `/usr/local` because `/home/iojs` is bound as a volume out of the container.